modules/product/src/test/java/com/opengamma/strata/product/swap/IborRateCalculationTest.java

```diff
@@ -889,6 +901,69 @@ public void test_expand_gearingSpreadEverythingElse() {
     assertThat(periods).containsExactly(rap1, rap2, rap3);
   }
 
+  //-------------------------------------------------------------------------
+  @Test
+  public void test_createAccrualPeriodsCny1wFixingStub() {
+    BusinessDayAdjustment businessDayAdjustment = BusinessDayAdjustment.of(FOLLOWING, SAT_SUN);
+    TenorAdjustment tenorAdjustment =
+        TenorAdjustment.of(Tenor.TENOR_1W, PeriodAdditionConventions.NONE, businessDayAdjustment);
+
+    ImmutableIborIndex index = ImmutableIborIndex.builder()
+        .currency(Currency.CNY)
+        .dayCount(ACT_365F)
+        .name("CNY-REPO-1W")
+        .fixingCalendar(SAT_SUN)
```

Comment:
No idea what happens if there is a holiday. Maybe the test case should cover that?